controllers/rack.go

```diff
@@ -379,7 +387,7 @@ func (r *SingleClusterReconciler) upgradeOrRollingRestartRack(found *appsv1.Stat
 			return found, reconcileError(nErr)
 		}
 
-		if needRollingRestartRack || len(failedPods) != 0 {
+		if needRollingRestartRack {
```

Comment:
If user update the CR back to original value to recover the cluster, in that case spec and status will be same. in that case will 'r.rollingRestartRack' get triggered?

Reply:
No, it won't. Now I remember why we were restarting failed pods always...

Reply:
Added it after scaleup